server/config/topic_page/equity/USA.textproto

```diff
@@ -120,6 +120,26 @@ blocks {
   }
 }
 
+blocks {
+  title: "Income inequality between sexes"
+  description: "Income inequality is computed as the ratio of the difference between median income for each sex, with the combined median incomes. A value of 0 means complete equality of median income."
+  stat_var_metadata {
+    stat_var: "GenderIncomeInequality_Person_15OrMoreYears_WithIncome"
```

Comment:
I think this needs refresh... data is from 2018 :(

Reply:
oh, let me try to run that pipeline...

Reply:
sorry I didn't mean it as blocking this change, but something to refresh